services/system/ProducerSys/include/services/system/ProducerSys.hpp

```diff
@@ -14,6 +14,13 @@ namespace SystemService
       static constexpr auto service      = psibase::AccountNumber("producer-sys");
       static constexpr auto serviceFlags = psibase::CodeRow::allowWriteNative;
 
+      // An action authorized by this account must be authorized by at
+      // least one honest producer
+      static constexpr auto producerAccountWeak = psibase::AccountNumber("prods-weak");
+      // If two actions are authorized by this account, there is at least
+      // one honest producer that authorized both actions.
+      static constexpr auto producerAccountStrong = psibase::AccountNumber("prods-strong");
```

Comment:
I don't know what you mean by "honest" in these comments.

Reply:
An honest producer is one that follows the rules, written or unwritten, for the chain and the actions being authorized. What that means is going to be dependent on the specific actions.  For system services it mostly means not breaking the chain.

Reply:
I think you're using the assumption that there are some minimum number of honest producers according to the consensus rules, but I think most people will read this and just think, "how do you know they're honest?". 

Also, the comment over checkAuthSys should be updated since it's no longer just ">2/3rds active producers"